services/graphql/src/definitions/config.js

```diff
@@ -11,6 +11,7 @@ module.exports = gql`
     contactUrl: String
     contactText: String
     leadershipEnabled: Boolean!
+    leadershipCompanyLabel: String!
```

Comment:
```suggestion
    leadershipCompanyLabel: String
```
This label shouldn't be required